vulture/core.py

```diff
@@ -143,7 +145,12 @@ def __hash__(self):
 class Vulture(ast.NodeVisitor):
     """Find dead code."""
 
-    def __init__(self, verbose=False):
+    def __init__(self, verbose=False, ignore_names=[]):
+        for name in ignore_names:
+            if any(fnmatchcase(unsafe_name, name)
+                    for unsafe_name in UNSAFE_IGNORE_NAMES):
+                raise ValueError(
+                        'It is not safe to ignore the name "{}".'.format(name))
```

Comment:
I think it would be better if we could just ignore the `ignore_names` list when checking for the string "property". We probably habe to test for "property" before deciding whether we cant to ignore the property.